dotcom-rendering/scripts/deploy/riff-raff.yaml

```diff
@@ -23,14 +23,14 @@ deployments:
   rendering:
     type: autoscaling
     parameters:
-      bucket: aws-frontend-artifacts
+      bucketSsmKey: /account/services/dotcom-artifact.bucket
     dependencies:
       - frontend-static
       - frontend-cfn
   frontend-static:
     type: aws-s3
     parameters:
-      bucket: aws-frontend-static
+      bucketSsmKey: /account/services/dotcom-static.bucket
```

Comment:
Don't we also need this? `bucketSsmLookup: true` 
We've prepared one similar PR for frontend: https://github.com/guardian/frontend/pull/25708

Reply:
I believe that's the default if you specify the key (at least it should be).

Reply:
Yes you're right 👍 I re-read Akash's email

Reply:
Adding a comment here just for reference: The above change was reverted as the `aws-s3` type doesn't support `bucketSsmKey` yet. See [here](https://github.com/guardian/frontend/pull/25708/files#r1030592952).